src/test/test_conformance.cpp

```
@@ -6,8 +6,8 @@
 
 #define CONFORMANCE_TEST_PATH "external/bpf_conformance/tests/"
 
-void test_conformance(std::string filename, bpf_conformance_test_result_t expected_result,
-                      std::string expected_reason) {
+static void test_conformance(std::string filename, bpf_conformance_test_result_t expected_result,
```

cppcheck

[performance] Function parameter 'filename' should be passed by const reference.

(passedByValue)